### PR TITLE
Campaign card v2

### PR DESCRIPTION
**Pull request by val-fcn**

### CODE REVIEW

**Review by TheoBoschetti — DISMISSED** [review]
snE

File: frontend/src/components/CampaignCard.jsx [hunk 1]
```diff
@@ -1,25 +1,75 @@
...
+          </a>
+        </div>
       </Link>
     </div>
   );
 }
 
 export default CampaignCard;
+
+/* TODO => faire un datediff en back à récuperer pour le comparer
```
Comment (LoicBrassart) [inline frontend/src/components/CampaignCard.jsx]: Nope, you don't tell future-you what he should/will have to do. Or at least, not here: that's whhat the backlog is for !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Delete this TODO please, after that we're clear ;-)

File: frontend/src/components/CampaignCard.jsx (hunk 1 above)
Comment (val-fcn) [reply frontend/src/components/CampaignCard.jsx]: Done !

**Review by TheoBoschetti — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
No More Evil ! \o/

**Pull request merged**